windows_containers/creating_windows_machinesets/creating-windows-machineset-aws.adoc

```diff
@@ -14,12 +14,27 @@ You can create a Windows `MachineSet` object to serve a specific purpose in your
 * You installed the Windows Machine Config Operator (WMCO) using Operator Lifecycle Manager (OLM).
 * You are using a supported Windows Server as the operating system image.
 +
-Use the following `aws` command to query valid AMI images:
+Use one of the the following `aws` commands, as appropriate for your Windows Server release, to query valid AMI images:
```

Comment:
Is this a Procedure? This particular step doesn't have a bullet so I wonder if it should be bulleted as a third prereq, or if it is a one-step procedure that follows the two prereqs that precede it?

Reply:
It is a command to determine if your image meets the associated pre-req. I think we are OK leaving that part as-is. Good thought, though.